docs/tutorials/12_deuteron_binding_energy.ipynb

```diff
@@ -0,0 +1,1286 @@
+      "text/plain": [
+       "<Figure size 864x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(12, 8))\n",
+    "plt.rcParams.update({\"font.size\": 22})\n",
+    "plt.plot(plots[2][0], plots[2][1], \"o-\", label=\"COBYLA\")\n",
+    "plt.axhline(\n",
+    "    y=reference_values[2],\n",
+    "    color=\"k\",\n",
+    "    linestyle=\"--\",\n",
+    "    label=\"Reference Value: {}\".format(reference_values[2]),\n",
+    ")\n",
+    "plt.legend()\n",
+    "plt.xlabel(\"Cost Function Evaluations\")\n",
+    "plt.ylabel(r\"$\\langle H_3 \\rangle $ - Energy (MeV)\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "08e784bf",
+   "metadata": {},
+   "source": [
+    "Note that with the quantum circuits given in [[1]](https://arxiv.org/pdf/1801.03897.pdf), we can't test whether we can reach the reference value for $H_4$, because we don't have an ansatz for this Hamiltonian. Fortunately, that is not the end for our experiments, because Qiskit provides some functions that create ansatz for us. For our purposes, we decided to use the TwoLocal function to build our new ansatz. In the code cell below we used a for loop to create a list with the TwoLocal ansatz for each Hamiltonian. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "c0da90ac",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "twolocal = []\n",
+    "for i in range(1, 5):\n",
+    "    ansatz_tl = TwoLocal(\n",
+    "        deuteron_hamiltonians[i - 1].num_qubits,\n",
+    "        [\"rz\", \"ry\"],\n",
+    "        \"cx\",\n",
+    "        entanglement=\"full\",\n",
+    "        reps=i,\n",
```

Comment:
Yes, it is necessary, because without it we can't achieve the reference value using the setup that I defined in `TwoLocal`. However, I believe that there is another way to do it without scaling `reps` with the size of the Hamiltonian.